articles/machine-learning/preview/how-to-use-model-data-collection.md

```diff
@@ -34,6 +34,12 @@ Then run the following command:
 
     pip install azureml.datacollector
 
+## Set environment variables
+
+Model data collection depends on two environment variables. AML_MODEL_DC_STORAGE_ENABLED must be set to true (all lowercase) and AML_MODEL_DC_STORAGE must be set to the connection string for the Azure Storage account where you want to store the data.
```

Comment:
Would be helpful if you put quotes around "true" to indicate it's a literal value.